packages/calling-stateful-client/src/CallContext.ts

```diff
@@ -625,20 +653,36 @@ export class CallContext {
     });
   }
 
-  public setDeviceManagerUnparentedView(
-    localVideoStream: LocalVideoStreamState,
-    view: VideoStreamRendererViewState | undefined
-  ): void {
+  public setDeviceManagerUnparentedView(args: {
+    localVideoStream: LocalVideoStreamState;
+    view: VideoStreamRendererViewState | undefined;
+    /* @conditional-compile-remove(video-background-effects) */
```

Comment:
is it because of conditional compilation that we wrap it using an args?

Reply:
nope, just many positional arguments, some of which may be undefined so I figured it made sense to make them named arguments now. Let me know if you want to revert back to positional args

Reply:
oh turns out it was for conditional comp lol, so i guess both reasons haha

Reply:
With the move of logic to internalcontext this change was removed - resolving!